source/macos/AdaptiveCards/AdaptiveCards/Views/ACRTextView.swift

```diff
@@ -87,6 +87,22 @@ class ACRTextView: NSTextView, SelectActionHandlingProtocol {
         }
     }
     
+    func customizeForFactSetCell(isTitleView: Bool, config: RenderConfig) {
+        setContentCompressionResistancePriority(.required, for: .vertical)
+        alignment = .left
+        isEditable = false
+        backgroundColor = .clear
+        if isTitleView {
+            let fontManager = NSFontManager.shared
+            font = fontManager.convert(font ?? NSFont.systemFont(ofSize: config.inputFieldConfig.font.pointSize), toHaveTrait: NSFontTraitMask.boldFontMask)
+        }
+        linkTextAttributes = [
+            NSAttributedString.Key.foregroundColor: config.hyperlinkColorConfig.foregroundColor,
+            NSAttributedString.Key.underlineColor: config.hyperlinkColorConfig.underlineColor,
+            NSAttributedString.Key.underlineStyle: config.hyperlinkColorConfig.underlineStyle.rawValue
```

Comment:
```suggestion
            .foregroundColor: config.hyperlinkColorConfig.foregroundColor,
            .underlineColor: config.hyperlinkColorConfig.underlineColor,
            .underlineStyle: config.hyperlinkColorConfig.underlineStyle.rawValue
```
Guess these can be ignored?